config/jobs.yaml

```diff
@@ -102,6 +102,7 @@ _anchors:
     - 'tip'
     - 'ulfh'
     - 'vireshk'
+    - 'linux-pci'
```

Comment:
I know _some_ Team Members who are strict about alphabetical order but I'm not one of them ;)

It will need sorting at some point anyway with "qcom" at the very top

Reply:
@pawiecz, I noticed this too, but I wasn't sure whether this was something we cared about or not.

Happy to make mends here in the follow-up change. :+1:

Reply:
Closing the loop with the following:

- https://github.com/kernelci/kernelci-pipeline/pull/1109